ee/ucp/admin/configure/route-reflector-deployment.md

```diff
@@ -0,0 +1,210 @@
+# Steps to deploy Route Reflector on UCP 3.0
+
+Note: If you are using UCP 3.0 on Azure deploying route reflectors is not required. The networking control plane is handled
+by Azure SDN and not calico.
+
+1)  On one of the nodes in Docker EE cluster, download the calicoctl binary from 
+    ```
+    curl -O -L https://github.com/projectcalico/calicoctl/releases/download/v3.1.1/calicoctl
+    chmod +x calicoctl
+    $PATH=$PATH:/<directory when calicoctl is downloaded>
```

Comment:
If I'm an admin, I'll probably need to use `calicoctl` more times. So consider moving the binary to `/usr/local/bin`.